src/Exersise1/UserService.java

```diff
@@ -0,0 +1,36 @@
+package Exersise1;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class UserService {
+
+    protected final static List<User> users = Arrays.asList(
```

Comment:
jeśli używamy zmiennych final static to nazwy zmiennych powinnismy pisać z dużych liter, czyli -> USERS :)